ReadASCIIApp/src/ReadASCII.cpp

```diff
@@ -576,61 +562,269 @@ void ReadASCII::readFilePoll(void)
     }
 }
 
-asynStatus ReadASCII::readFile(const char *dir)
+asynStatus ReadASCII::readFile(const char *dir, bool retry)
 {
```

Comment:
The function no longer sets `fileBad = true` on error - just checking that it was no longer required